exp/views/study.py

```diff
@@ -2097,12 +2098,11 @@ def get_csv_demographic_row_and_headers(self, resp=None):
             (
                 "demographic_child_rounded_ages",
                 round_ages_from_birthdays(
```

Comment:
So, instead of using `DemographicSnapshot.created_at` we're using `Response.date_created`? Can you clarify how this solves the problem, seeing as both are `models.DateTimeField(auto_now_add=True)` (in other words, auto-populated)?